platforms/hanwha-wn/CMakeLists.txt

```diff
@@ -57,4 +57,3 @@ set (Boost_INCLUDE_DIRS ${CMAKE_INSTALL_PREFIX}/include)
 set (Boost_LIBRARY_DIRS ${CMAKE_INSTALL_PREFIX}/lib)
 
 buildSdk()
-
```

Comment:
But as far as I understand, one can always make a trivial pull request consisting solely of cosmetic changes and it should be trivial to have an approval.
I am still strongly opposed to piggybacking of cosmetic fixes, no matter how small, to other commits.